src/AdafruitIO.cpp

```diff
@@ -165,10 +165,21 @@ const __FlashStringHelper* AdafruitIO::statusText()
   }
 }
 
-void AdafruitIO::run(uint16_t busywait_ms)
+aio_status_t AdafruitIO::run(uint16_t busywait_ms)
 {
+  uint32_t timeStart = millis();
+  aio_status_t net_status = networkStatus();
+  // If we aren't connected, return network status -- fail quickly
+  // Previous version would just hang on a network error
+  if(net_status != AIO_NET_CONNECTED) {
+    _status = net_status;
```

Comment:
`_status` never instantiated? Due to change in L174, you may want to set to value of a call to `networkStatus()`

Reply:
Switched it to call status() instead of networkStatus().  Changed all _status to status() calls.

Reply:
I think the latest resolves all of these -- I'll leave it to you to push the buttons once the build test clears.